### ISSUE TITLE
deno fetch prints out compiler errors

### ISSUE THREAD

**Issue by kitsonk** [denoland/deno#2438]
If the code you are fetching has compiler/TypeScript errors, those get printed when you run `deno fetch`.  While obviously the code is run through the compiler to do the analysis of what needs to be fetched, it is still "strange" to have the errors logged.

After #2310 we should consider a better way of dealing with any compiler diagnostics.

**Comment by bartlomieju** [comment]
@kitsonk I think that showing output of TS compiler is fine in this case - after all you want to fetch and precompile code - I want to know if code is not compilable. Separate issue is naming `deno fetch` is definitely the most confusing name.

**Comment by kitsonk** [comment]
This hasn't aged well, closing.
